src/test/java/core/basesyntax/dao/FruitDaoTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.shop.item.Fruit;
+import org.junit.Test;
+
+public class FruitDaoTest {
+
+    private static final Fruit BANANA0 = new Fruit("banana", 0);
+    private static final Fruit BANANA30 = new Fruit("banana", 30);
```

Comment:
```suggestion
    private static final Fruit thirtyBananas = new Fruit("banana", 30);
```
для змінних класів краще використати звичний неймінг

Reply:
+